commands/http/handler.go

```diff
@@ -8,6 +8,8 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/rs/cors"
```

Comment:
needs to be vendored. run `make vendor` in project root.

you may have to `godep restore` or manually `go get` dependencies for it to work correctly, i believe.